File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/DeviceApiResponse.java
package com.onscreensync.tvapp.apiresponses;

import com.google.gson.annotations.SerializedName;

public class DeviceApiResponse {
    @SerializedName("deviceName")
    private String name;

    @SerializedName("tenantId")
    private String tenantId;

    @SerializedName("id")
    private String id;

    public String getName() {
        return name;
    }

    public String getTenantId() {
        return tenantId;
    }

    public String getId() {
        return id;
    }
}

File: src/app/src/main/java/com/onscreensync/tvapp/apiresponses/DeviceApiResponse.kt
package com.onscreensync.tvapp.apiresponses

import com.google.gson.annotations.SerializedName

data class DeviceApiResponse(
    @SerializedName("deviceName")
    val name: String?,
    @SerializedName("tenantId")
    val tenantId: String?,
    @SerializedName("id")
    val id: String?
)
